python/ray/rllib/example.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+"""
+This demonstrates the RLlib algorithm API through a simple bakeoff.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import ray
+import ray.rllib.evolution_strategies as es
+import ray.rllib.policy_gradient as pg
+
+
+if __name__ == '__main__':
```

Comment:
Let's use `"` everywhere instead of `'`.

I'm open to doing a project wide change to `'` if you prefer.